3.component_model.md

```diff
@@ -286,7 +286,7 @@ See the Units section above for valid values.
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `required` | `string` | Y | | The minimum amount of memory in MB required  for running this container. The value should be a positive integer, greater than zero. |
+| `required` | `string` | Y | | The minimum amount of memory required  for running this container. The value should be a positive integer or positive integer using one of these suffixes: P, T, G, M, K. |
```

Comment:
Can you add a line saying the default would be "bytes" if user dismiss any unit?

Reply:
what about your options?
1. The minimum amount of memory required in bytes for running this container. The value should be a positive integer or positive integer using one of these suffixes: P, T, G, M, K.

2. The minimum amount of memory required for running this container. The value should be a positive integer or positive integer using one of these suffixes: P, T, G, M, K. The default unit is bytes if no suffix.

Reply:
second one modified:
```
The minimum amount of memory required for running this container. The value should be a positive integer with/without unit suffix: P, T, G, M, K. If no unit is given, it defaults to 'bytes'.
```

Reply:
done